android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -921,6 +921,9 @@ private static void copyAssets(Context context) {
       copyAsset(context, KMFilename_KmwCss, "", true);
       copyAsset(context, KMFilename_KmwGlobeHintCss, "", true);
       copyAsset(context, KMFilename_Osk_Ttf_Font, "", true);
+      
+      // Needed until our minimum version of Chrome is 61.0+.
+      copyAsset(context, KMFilename_JSPolyfill2, "", true);
```

Comment:
This corresponds to the KeymanWeb artifact
https://github.com/keymanapp/keyman/blob/486807fd7d3fe09390154837648093331f85f921/android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java#L356

Does the KMEA/build.sh section copy this? (I'm not sure I see it)
https://github.com/keymanapp/keyman/blob/486807fd7d3fe09390154837648093331f85f921/android/KMEA/build.sh#L76-L90

Reply:
It doesn't need to.  It's checked in.

https://github.com/keymanapp/keyman/blob/486807fd7d3fe09390154837648093331f85f921/android/KMEA/app/src/main/assets/other-polyfills.js

Reply:
ah, that's where it was